library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -533,3 +610,45 @@ impl fmt::Display for ExitStatus {
         }
     }
 }
+
+#[unstable(feature = "linux_pidfd", issue = "none")]
+#[derive(Debug)]
+pub struct PidFd {
+    fd: libc::c_int,
+}
+
+#[unstable(feature = "linux_pidfd", issue = "none")]
+impl crate::os::unix::io::AsRawFd for PidFd {
+    fn as_raw_fd(&self) -> crate::os::unix::io::RawFd {
+        self.fd
+    }
+}
+
+#[unstable(feature = "linux_pidfd", issue = "none")]
+impl crate::os::unix::io::IntoRawFd for PidFd {
```

Comment:
What would be the use for that? There are currently no methods on the `PidFd` type.

Reply:
1. Because a potential crate that extends functionality of a pidfd (for example a kill crate) would probably want to pass an pidfd around instead of a raw fd, so that the dropping is handled by std.
2. Because as a user of std I would expect that `FromRawFd` is implemented when `IntoRawFd` is implemented - they kind of belong together in my mind.

I am also fine with not doing it - but that should be a conscious decision and not an oversight :)

Reply:
I'm fine with adding it for symmetry's sake.